proposals/0029-cooperative-vector.md

```diff
@@ -142,7 +142,7 @@ declare <[NUMo] x [TYo] @dx.op.matvecmul.v[NUMo][TYo].v[NUMi][TYi](
     immarg i32,       ; matrix layout
     immarg i1,        ; matrix transpose
```

Comment:
I think the matrix transpose property should be specified through the operation flags as well.